src/Event.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
```

Comment:
Did you add this project to test in Unity yet? It should generate `.meta` files that will need to be added to the repo.

Reply:
Ah hmm I can get it to generate these files